libblkid/src/Makemodule.am

```diff
@@ -50,6 +50,7 @@ libblkid_la_SOURCES = \
 	libblkid/src/superblocks/bitlocker.c \
 	libblkid/src/superblocks/bluestore.c \
 	libblkid/src/superblocks/btrfs.c \
+	libblkid/src/superblocks/cs_fvault2.c \
```

Comment:
The same for meson?

Reply:
Weird that the tests passed without the meson definitions.

Reply:
No tests actually ran...

Reply:
ah, meson, thanks, completely forgot about it.

Reply:
should be fixed.